### PR TITLE
feat: added docs for the mainnet

### PR DESCRIPTION
**Pull request by Mikelle**

### CODE REVIEW

**Review by aloknerurkar — APPROVED** [review]

File: v0.8.0/developers/mainnet.mdx [hunk 1]
```diff
@@ -3,7 +3,91 @@ title: Mev-commit Mainnet
...
+  </thead>
+  <tbody>
+    <tr>
+      <td>SettlementGateway</td>
+      <td>{SettlementGatewayMainnet0_8_0}</td>
+    </tr>
+  </tbody>
+</table>
+
+### **Bridge Gateway (Holesky)**
```
Comment (shaspitz) [inline v0.8.0/developers/mainnet.mdx:73]:
```suggestion
### **Bridge Gateway (L1 Ethereum)**
```

File: v0.8.0/developers/contracts.mdx [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 The [mev-commit contracts](https://github.com/primev/mev-commit/tree/main/contracts) verify and manage the movement of funds as a response to bid and commitment activities that occur on our p2p network.
 
+For mev-commit mainnet contract addresses please refer to [Mainnet](/v0.8.0/developers/mainnet) section.
```
Comment (mrekucci) [inline v0.8.0/developers/contracts.mdx:5]: This should be `v1.0.0`.

File: v0.8.0/developers/contracts.mdx [hunk 3]
```diff
@@ -2,6 +2,8 @@
...
+
```
Comment (mrekucci) [inline v0.8.0/developers/contracts.mdx:6]: This should be `v1.0.0`.

File: v0.8.0/developers/mainnet.mdx [hunk 4]
```diff
@@ -80,3 +164,11 @@ import {
...
 
 All contract calls should be made through the proxy contracts. Implementations are listed for transparency, where verified contract code can be viewed.
+
+## How can users provide feedback or report issues encountered on the testnet?
+
+Users can open an issue on the GitHub repository that is associated with any issue they encounter. They can also tag @primev_xyz on Twitter, or reach out to us to join a Telegram group for early power users.
+
+## Mainnet System Requirements
+
+Please refer to the System Requirements section [here.](/v0.8.0/developers/system-requirements)
```
Comment (mrekucci) [inline v0.8.0/developers/mainnet.mdx:174]: This should be `v1.0.0`.

File: v0.8.0/get-started/validators/validator-guide.mdx [hunk 5]
```diff
@@ -31,7 +31,8 @@ By opting-in to the mev-commit protocol as a validator **in one of the three for
 | Relay | Docs |
 |-------|------|
 | Titan | [docs.titanrelay.xyz](https://docs.titanrelay.xyz/) |
-| Aestus| [holesky.aestus.live](https://holesky.aestus.live/) |
+| Aestus Mainnet | [mainnet.aestus.live](https://mainnet.aestus.live/) |
+| Aestus Holesky | [holesky.aestus.live](https://holesky.aestus.live/) |
 
 We expect all known relays to support opting-in to mev-commit within a few months of being live on mainnet. If you are a relay looking to join the mev-commit network please visit our [Relays page](/v0.8.0/get-started/relays) to get more information.
```
Comment (mrekucci) [inline v0.8.0/get-started/validators/validator-guide.mdx:37]: This should be `v1.0.0`.

**Pull request merged**